src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -446,6 +446,17 @@
         "RLIKE",
         Sequence("ILIKE", Ref.keyword("ANY", optional=True)),
         "REGEXP",
+    ),
+     ComparisonOperatorGrammar=OneOf(
```

Comment:
One too many spaces here:

```suggestion
    ComparisonOperatorGrammar=OneOf(
```